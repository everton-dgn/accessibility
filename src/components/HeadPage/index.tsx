import Head from 'next/head'

type TitleProps = {
  title: string
}

const HeadPage = ({ title }: TitleProps) => (
  <Head>
    <meta charSet="utf-8" />
    <meta
      name="viewport"
      content="minimum-scale=1, initial-scale=1, width=device-width"
    />
    <title>{title}</title>
  </Head>
)

export default HeadPage
