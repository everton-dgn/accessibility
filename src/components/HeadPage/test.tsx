import { renderWithTheme } from 'utils/tests/helpers'
import HeadPage from '.'

describe('<HeadPage />', () => {
  it('should render a snapshot', () => {
    const { container } = renderWithTheme(<HeadPage title="Acessibilidade" />)

    expect(container).toMatchSnapshot()
  })
})
