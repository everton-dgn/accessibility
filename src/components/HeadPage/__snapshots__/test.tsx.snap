// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeadPage /> should render a snapshot 1`] = `<div />`;
